qa-covid19.planx-pla.net/manifest.json

```diff
@@ -0,0 +1,139 @@
+{
+  "notes": [
+    "This is the dev environment manifest",
+    "That's all I have to say"
+  ],
+  "versions": {
+    "arborist": "quay.io/cdis/arborist:2020.03",
+    "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
+    "fence": "quay.io/cdis/fence:2020.03",
+    "indexd": "quay.io/cdis/indexd:2020.03",
+    "peregrine": "quay.io/cdis/peregrine:2020.03",
+    "pidgin": "quay.io/cdis/pidgin:2020.03",
+    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
+    "sheepdog": "quay.io/cdis/sheepdog:4.0.0",
+    "portal": "quay.io/cdis/data-portal:2.24.8",
+    "tube": "quay.io/cdis/tube:2020.03",
+    "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
+    "spark": "quay.io/cdis/gen3-spark:2020.03",
+    "hatchery": "quay.io/cdis/hatchery:2020.03",
+    "wts": "quay.io/cdis/workspace-token-service:2020.03",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.03",
+    "ambassador": "quay.io/datawire/ambassador:0.60.3"
+  },
+  "global": {
+    "environment": "qa-covid19",
+    "hostname": "qa-covid19.planx-pla.net",
+    "revproxy_arn": "arn:aws:acm:us-east-1:236714345101:certificate/ec368910-95e7-4f7f-bdd3-020a2f51bc94",
```

Comment:
This should be the same as qaplanetv2: "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8"